util/settings/settings.go

```diff
@@ -63,6 +63,12 @@ type ArgoCDSettings struct {
 	HelmRepositories []HelmRepoCredentials
 	// AppInstanceLabelKey is the configured application instance label key used to label apps. May be empty
 	AppInstanceLabelKey string
+	// ResourceOverrides holds the overrides for specific resources
+	ResourceOverrides map[string]ResourceOverride
```

Comment:
Can you add a comment explaining keys are group/Kinds, and give an example key (e.g. argoproj.io/Rollouts)

Reply:
done

Reply:
I noticed rollout is lowercase here. We should keep Kind to match the exact casing of the k8s kind (e.g. `argoproj/Rollouts`), which I believe is significant in kubernetes, and even for us. This means the `resource_customizations/argoproj.io/rollouts` should be renamed to `resource_customizations/argoproj.io/Rollouts`. Matching case will prevent issues down the road since we can trust == string comparisons instead of always having to do something like strings.ToLower().

Reply:
That makes sense and I can make that change. However, can you confirm you mean `Rollout` instead of `Rollouts`?